packages/core/style.css

```diff
@@ -287,13 +265,32 @@ body {
 
 body {
   font-size: var(--root-font-size);
-  /* background-color: var(--bg-gray-01); */
   color: #333;
   margin: 0;
   box-sizing: border-box;
-  font-family: var(--main-text-font), -apple-system, BlinkMacSystemFont,
-    "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji",
-    "Segoe UI Emoji", "Segoe UI Symbol";
+
+  /**
+   * On Linux, Firefox consults a third-party tool called fontconfig when
+   * deciding which fonts to display. fontconfig maps the names of proprietary
+   * fonts (e.g., Helvetica) to similar but non-identical free fonts (e.g.,
+   * Nimbus Sans), which Firefox uses instead.[1][2]
+   *
+   * This particular mapping is a problem for us. Nimbus Sans is not a great
+   * replacement for Helvetica because it has an em-box which leaves a large
+   * amount of empty space below the baseline. This causes text not to appear
+   * vertically-centered, which is especially problematic in the OptionMenu
+   * component.
+   *
+   * fontconfig does not map the name Helvetica Neue to Nimbus Sans, so it's a
+   * better choice than Helvetica here. Arial has a reasonable replacement and
+   * the default sans-serif font is also fine.
+   *
+   * [1] https://nolanlawson.com/2020/05/02/customizing-fonts-in-firefox-on-linux/
+   * [2] https://bugzilla.mozilla.org/show_bug.cgi?id=553580
+   */
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
+    "Segoe UI Symbol";
```

Comment:
Another option is to just copy the antd font stack:

```css
font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
```

That would solve the same problem and maintain compatibility with ant, which is probably a good rule of thumb, but *Fira Sans* would be used preferentially if installed on Mac.